packages/manager/modules/exchange/src/account/alias/account-alias.html

```diff
@@ -13,7 +13,12 @@
             <oui-datagrid-column
                 data-title="'exchange_tab_ALIAS_header' | translate"
                 data-property="displayName"
-            ></oui-datagrid-column>
+            >
+                <div
+                    class="account-alias-column text-wrap"
+                    data-ng-bind="$value"
+                ></div>
```

Comment:
I think we must have the P element to display the text.
```suggestion
                <div
                    class="account-alias-column text-wrap"
                ><p data-ng-bind="$value"></p></div>
```